pymeasure/experiment/results.py

```diff
@@ -142,10 +144,9 @@ def format(self, record):
         :return: a string
         """
         line = []
-        numeric_types = (float, int, Decimal)
         for x in self.columns:
             value = record.get(x, float("nan"))
-            if type(value) in numeric_types:
+            if type(value) in self.numeric_types:
```

Comment:
Why not `isinstance(value, numbers.Number)` instead of dragging around a class attribute?

Reply:
Good idea, fixed in the last commit.

Reply:
@bilderbuchi it's been a while since this PR, I just rediscovered why I use `type(value) in self.numeric_types`.

`isinstance(True, numbers.Number)` evaluates to `True`, which means if a boolean is passed it would be interpreted as a data point 1. [This test]( https://github.com/mcdo0486/pymeasure/blob/6b33d02d69ae6cbe526456b928caa1324132581a/tests/experiment/test_results.py#L95) catches that case.

Reply:
Ok, sorry, I didn't pick that up.